paddle/cinn/ir/tensor.cc

```diff
@@ -52,6 +52,23 @@ Tensor _Tensor_::Make(const std::string &name,
 
   return Tensor(n);
 }
+Tensor _Tensor_::Make(const std::string &name,
+                      Type dtype,
+                      const std::vector<Expr> &shape,
+                      const std::vector<Expr> &domain,
+                      const std::vector<Var> &reduce_axis) {
+  CHECK(!name.empty()) << "Tensor name is set empty";
```

Comment:
"Cannot set empty Tensor name in _Tensor_::Make"

Reply:
Done